src/web3/saga.ts

```diff
@@ -61,8 +60,13 @@ export function* getOrCreateAccount() {
     Logger.debug(TAG + '@getOrCreateAccount', 'Creating a new account')
 
     const mnemonicBitLength = MnemonicStrength.s128_12words
-    const mnemonicLanguage = getMnemonicLanguage(yield* select(currentLanguageSelector))
-    let mnemonic: string = yield* call(generateMnemonic, mnemonicBitLength, mnemonicLanguage, bip39)
+    const mnemonicLanguage = MnemonicLanguages.english
+    let mnemonic: string = yield* call(
+      generateMnemonic,
+      mnemonicBitLength,
+      MnemonicLanguages.english,
```

Comment:
Can this just be set to `mnemonicLanguage`? since it is set as english in the lines above.

Reply:
Fixed in b9d9e56c4f7f8219b05fdd5ad152a918146c1835!